libs/zauth/test/Arbitraries.hs

```diff
@@ -42,14 +48,20 @@ instance Arbitrary Bot where
 instance Arbitrary Provider where
     arbitrary = mkProvider <$> arbitrary
 
+instance Arbitrary LegalHoldAccess where
+    arbitrary = mkLegalHoldAccess <$> arbitrary <*> arbitrary
+
+instance Arbitrary LegalHoldUser where
+    arbitrary = mkLegalHoldUser <$> arbitrary <*> arbitrary
+
 instance Arbitrary ByteString where
     arbitrary = fromString <$> arbitrary `suchThat` (not . any (== '.'))
 
 instance Arbitrary Signature where
     arbitrary = Signature <$> arbitrary
 
 instance Arbitrary Type where
-    arbitrary = elements [A, U]
+    arbitrary = elements [A, U, LA, LU]
```

Comment:
```suggestion
    arbitrary = elements [minBound..]
```

(That requires deriving Bounded, Enum in `Type`, and it's also not the same.  But why are some constructors missing?  Would that break anything?  Or fix anything?)